app/controllers/advertisements_controller.rb

```diff
@@ -0,0 +1,28 @@
+class AdvertisementsController < ApplicationController
+  def index
+    @advertisement = Advertisement.all
+  end
+
+  def show
+    @advertisement = Advertisement.find(params[:id])
+  end
+
+  def new
+    @advertisement = Advertisement.new
+  end
+
+  def create
+    @advertisement = Advertisement.new
```

Comment:
This is how they gave us the code in `posts_controller.rb` which is what this controller is based off of. Is there a reason you would want it to not be an instance variable?